android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java

```diff
@@ -62,6 +63,26 @@
   private Activity currentActivity;
   private static final Pattern animationIdentifierPattern = Pattern.compile("^.+:.+/");
 
+  public Integer setColor(CustomTabsIntent.Builder builder, final ReadableMap options, String key, String method, String colorName) {
+    String colorString = "";
```

Comment:
can be `String colorString = options.getString(key);`?

and further on try/catch block : 

```
String colorString = options.getString(key);

...

try {
if (colorString != null) {
...
}

...
```

Reply:
We need to validate if the key exist first, using `options.hasKey(key)`

Reply:
@reberthkss did you find another bug?

Reply:
just for known more, the response from `getString` function is Nullable, so if the key doesn't exist the `colorString` variable was null, alright? Why don't follow this approach? Is it a bad practice/insecure?

Reply:
Oh no, this validation is old, if you want I can change it, no problem, thanks for noticing!

Reply:
Done!